src/chrome/manifest.json.mustache

```diff
@@ -2,14 +2,14 @@
   "name": "Hypothesis - Web & PDF Annotation",
   "short_name": "Hypothesis",
   "version": "{{ version }}",
-  "version_name": "{{ version }} ({{ version_name }})",
+  "version_name": "{{ version }} ({{ versionName }})",
   "manifest_version": 2,
 
-  {% if browser == 'chrome' %}
+  {{#browserIsChrome}}
   "minimum_chrome_version": "38",
-  {% endif %}
+  {{/browserIsChrome}}
 
-  {% if browser == 'firefox' %}
+  {{#browserIsFirefox}}
   "applications": {
     "gecko": {
```

Comment:
The Jinja comment tag in this section needs to be changed to a mustache comment tag, otherwise the result is invalid JSON. Apart from that, loading the Firefox add-in via the "Load Temporary Add-on" button at `about:debugging#addons` worked.